docs/source/conf.py

```diff
@@ -368,7 +368,7 @@
             'ket': ["\\left| #1 \\right\\rangle", 1],
             'bra': ["\\left\\langle #1 \\right|", 1],
             'braket': ["\\left\\langle #1 | #2 \\right\\rangle", 2],
-            'vec': ["\\text{vec}\\left(#1\\right)", 1],
+            'vect': ["\\text{vec}\\left(#1\\right)", 1],
```

Comment:
N.B.: this is new in the latest diff. Copy/pasting commit message:

```
Rename custom \vec macro to \vect

This was shadowing the built-in latex/mathjax macro \vec{}, which
seems like a bad idea and made for ugly equations in intro.rst, which
(surprisingly) is the only place \vec{} was being used.
```

The other option would be to just drop this macro, since it isn't actually used anywhere, but then the same could be said of most of the rest of the macros.  Only `\bra`, `\ket`, and `\tr` are currently used, as far as I can tell...

Reply:
Here are screenshots demonstrating the difference.

### Default `\vec{}` (after this change)

<img alt="default vec" src="https://drive.google.com/uc?id=1PhD6T8TPQVvfcILTWy66f1mq9EBB5CUi">

https://drive.google.com/open?id=1PhD6T8TPQVvfcILTWy66f1mq9EBB5CUi

### Custom `\vect{}` (before this change)

<img alt="custom vect" src="https://drive.google.com/uc?id=1-LzWyRrUfU2K4tGqRKCcJO-NcYirAeNW">

https://drive.google.com/open?id=1-LzWyRrUfU2K4tGqRKCcJO-NcYirAeNW

Reply:
Also note that the default "overarrow" vector rendering is what was used prior to the recent merge of #925.